cloud/gcp/storage/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "region" {
+  type        = string
+  description = "Default region for bucket resources"
+}
+
+variable "company-name" {
```

Comment:
suggestion (blocking): looks like the [existing TF code](https://github.com/seattle-uat/civiform/blob/main/cloud/azure/variables.tf) follows the convention of using `snake_case` rather than `kebab-case` for variable names. please update for consistency

Reply:
Done